foia_hub/newrelic.ini

```diff
@@ -33,7 +33,7 @@
 # application as you would like it to show up in New Relic UI.
 # The UI will then auto-map instances of your application into a
 # entry on your home dashboard page.
-app_name = openFOIA
+# app_name = In $NEW_RELIC_APP_NAME
```

Comment:
Yes, `newrelic.ini` takes precedence over the the env variables. I left the line in there to provide additional documentation that the name is in `$NEW_RELIC_APP_NAME`